cypress/integration/ui/accessibility.js

```diff
@@ -1,6 +1,7 @@
-describe('UI - Accessibility', function () {
+// eslint-disable-next-line mocha/no-skipped-tests
+describe.skip('UI - Accessibility', function () {
```

Comment:
As explained above, I skipped these rules only so we would pass the CI for now. You clearly commented why the CI didn't pass because of this accessibility tests but I didn't want to miss another reason for a CI failure on my PR. We should make a priority right after this PR to enable these tests again, disabling indeed specific rules too difficult to comply with and hopefully to retrieve a higher coverage. I just think it should be another PR to avoid delay on this one.

Reply:
Yes, good point for not missing other causes for CI failure with the extra noise. So the tests can be disabled for now, or if you prefer to be reminded (since I see they have since been reenabled), it'd be fine for them be failing in CI for the time-being now too--whatever works best for you. Glad to hear you're on board about selectively reeanbling and disabling problematic rules at some point instead.

Reply:
(Just updated my comment to mention that Travis is not currently running tests at all--it is only doing linting--so even test failures will not block CI, let alone thresholds. (I just wanted thresholds to fail so we'd notice when manually running tests, and for when we do reenable tests in CI in the future.).)